File: frontend/src/app/Models/Titular.ts
import { Endereco } from "./Endereco";

export interface Titular {
  id? : number;
  nome?: string | null;
  email?: string | null;
  cpf?: string | null;
  senha?: string | null;
  token?: string | null;
  endereco?: Endereco | null;
}

File: backend/Troopers.Capibank/Domain/Models/Titular.cs
namespace Troopers.Capibank.Domain.Models;

public class Titular
{
    public int Id { get; set; }
    public string? Nome { get; set; }
    public string Email { get; set; }
    public string CPF { get; set; }
    public string Senha { get; set; }
    public DateTime? CriadoEm { get; set;} = DateTime.Now;
    public DateTime? AlteradoEm { get; set; } = DateTime.Now;
    public Endereco? Endereco { get; set; }
    public IEnumerable<Atendimento> ListaAtendimentos = [];

}
